aten/src/ATen/native/TensorShape.cpp

```diff
@@ -18,13 +18,36 @@ static void check_cat_no_zero_dim(TensorList tensors) {
   }
 }
 
+// FIXME when empty tensors with arbitrary shapes is implemented
```

Comment:
can you add a little description about why having arbitrary shapes matter and what will change?  Just a simple example like we discussed in the previous PR.